ts/Series/CenteredUtilities.ts

```diff
@@ -78,7 +79,8 @@ namespace CenteredUtilities {
             plotWidth = chart.plotWidth - 2 * slicingRoom,
             plotHeight = chart.plotHeight - 2 * slicingRoom,
             centerOption: Array<(number|string|null)> = options.center as any,
-            smallestSize = Math.min(plotWidth, plotHeight);
+            smallestSize = Math.min(plotWidth, plotHeight),
+            thickness = options.thickness as number * 2 || {};
```

Comment:
This not necessarily is a `number` because `thickness` can be `undefined` in options. You should rather check whether it is a number and then fire the code you added.